src/services/addressResolver/index.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { ThirdPartyAPIEntryProps } from "../../common/hooks/useThirdPartyAPIEndpoints";
+
+export const resolveAddressNameByEndpoint = async (url: string) => {
+  try {
+    const response = await axios.get(url);
+    return response.data.identity.name;
+  } catch (e) {
+    return undefined;
```

Comment:
if you are going to discard error, the minimal thing is to log it. This saves you time later when you are chasing after bugs